components/demo/demo-snippet.js

```diff
@@ -34,10 +34,13 @@ class DemoSnippet extends LitElement {
 			}
 			.d2l-demo-snippet-demo {
 				flex: 1 1 auto;
-				padding: 18px;
+				overflow: hidden;
 				position: relative;
 			}
-			:host([no-padding]) .d2l-demo-snippet-demo {
+			.d2l-demo-snippet-demo-padding {
+				padding: 18px;
+			}
+			:host([no-padding]) .d2l-demo-snippet-demo-padding {
```

Comment:
Some tweaks here to the demo snippet so that things that would overflow get capped.

Reply:
These are breaking the floating buttons visual diffs (preventing them from working in the demo snippet). I'll look into this more.